python/pyspark/ml/classification.py

```diff
@@ -581,8 +602,11 @@ def _create_model(self, java_model):
 
 
 @inherit_doc
-class DecisionTreeClassificationModel(DecisionTreeModel, JavaMLWritable, JavaMLReadable):
+class DecisionTreeClassificationModel(DecisionTreeModel, JavaClassificationModel, JavaMLWritable,
```

Comment:
Just curious to know why we don't expose `numClasses` in `GBTClassificationModel`. Do we not support multiclass currently, or is there some other reason?